cmd/secret/cmd/root.go

```diff
@@ -0,0 +1,82 @@
+// Package cmd provides functions set and get with cobra library
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-itools-internship/go-secret/pkg/crypto"
+	"github.com/go-itools-internship/go-secret/pkg/io/storage"
+	"github.com/go-itools-internship/go-secret/pkg/provider"
+	"github.com/spf13/cobra"
+)
+
+type root struct {
+	key       *string
+	cipherKey *string
+	value     *string
+	path      *string
+	rootCmd   *cobra.Command
+}
+
+func (r *root) RootExecute(ctx context.Context) error {
+	err := r.rootCmd.ExecuteContext(ctx)
+	return err
+}
+
+func NewRoot() *root {
+	var rootCmd = &cobra.Command{
+		Use:   "root",
+		Short: "root includes cli commands",
```

Comment:
Try to find more descriptive text for describing what our tool is going todo. You could use readme.md for inspiration.